doc/FAQ

```diff
@@ -0,0 +1,101 @@
+- NodeOS - Lightweight operating system using Node.js as userspace - FAQ -
```

Comment:
Hahaha I'm alive, just bogged in work since school started. You'd like the wiki pages in the doc folder?